backend/module/authn/authn.go

```diff
@@ -92,9 +92,9 @@ func (a *api) Callback(ctx context.Context, request *authnv1.CallbackRequest) (*
 	// set the cookie header
 	// redirect back to original location
 	md := metadata.New(map[string]string{
-		"Location":         redirectURL,
-		"Location-Status":  "303",
-		"Set-Cookie-Token": token,
+		"Location":          redirectURL,
+		"Location-Status":   "303",
+		"Link-Cookie-Token": token,
```

Comment:
This looks like an unrelated change?

Reply:
whoops, was overzealous goland refactor. good catch.